File: vds-promotion/src/main/java/com/ttcn/promotionsdk/ui/theme/TabChipToken.kt
package com.ttcn.promotionsdk.ui.theme

import androidx.annotation.ColorInt

data class TabChipToken(
    @ColorInt val activeBackgroundColor: Int? = null,
    @ColorInt val inactiveBackgroundColor: Int? = null,
    @ColorInt val activeTextColor: Int? = null,
    @ColorInt val inactiveTextColor: Int? = null,
    /** Corner radius in **dp**. */
    val cornerRadius: Float? = null,
)

File: iosPromotionSDK/PromotionSDKUI/Theme/Token/TabChipToken.swift
//
//  TabChipToken.swift
//  PromotionSDK
//
//  Token theme. Đối ứng `token/TabChipToken.kt` bên Android — cùng field, cùng thứ tự.
//

import UIKit

/// Token cho tab dạng chip (vd "Tất cả / Sắp hết hạn" ở màn ưu đãi của tôi).
public struct TabChipToken {
    public var activeBackgroundColor: UIColor?
    public var inactiveBackgroundColor: UIColor?
    public var activeTextColor: UIColor?
    public var inactiveTextColor: UIColor?
    public var cornerRadius: CGFloat?

    public init(activeBackgroundColor: UIColor? = nil,
                inactiveBackgroundColor: UIColor? = nil,
                activeTextColor: UIColor? = nil,
                inactiveTextColor: UIColor? = nil,
                cornerRadius: CGFloat? = nil) {
        self.activeBackgroundColor = activeBackgroundColor
        self.inactiveBackgroundColor = inactiveBackgroundColor
        self.activeTextColor = activeTextColor
        self.inactiveTextColor = inactiveTextColor
        self.cornerRadius = cornerRadius
    }
}
